src/index.js

```diff
@@ -947,16 +972,19 @@ class Firestore extends commonGrpc.Service {
    * A funnel for read-only streaming API requests, assigning a project ID where
    * necessary within the request options.
    *
-   * The stream is returned in paused state and needs to be resumed once all
-   * listeners are attached.
+   * The Promise returned from this method resolves with callback that needs to
+   * be invoked in order to obtain the underlying stream. The stream stream is
+   * returned in paused state and should to be resumed once all listeners are
```

Comment:
stream stream => stream
should to be => should be

Given your callback/factory change thing, is it even necessary to pause / resume the stream anymore?

Reply:
It was not, but I didn't want to debug even more test failures...